src/h5web/providers/jupyter/JupyterProvider.tsx

```diff
@@ -10,7 +11,13 @@ interface Props {
 
 function JupyterProvider(props: Props) {
   const { url, filepath, children } = props;
-  const api = useMemo(() => new JupyterApi(url, filepath), [filepath, url]);
+  const api = useMemo(
+    () =>
+      process.env.REACT_APP_JLAB_DEV === 'true'
+        ? new JupyterDevApi(url, filepath)
+        : new JupyterApi(url, filepath),
+    [filepath, url]
+  );
```

Comment:
You could store the right constructor in a constant outside of the component completely, since it's a build-time thing. Like `const JupyterApi = process.env.REACT_APP_JLAB_DEV === 'true' ? JupyterDevApi : JupyterStableApi`